src/message/mod.rs

```diff
@@ -576,10 +572,15 @@ impl<'a> MessageFormatter<'a> {
             text.lines.push(Line::from(vec![leading, date, trailing]));
         }
 
+        let user_gutter_empty = std::iter::repeat(' ')
+            .take(self.settings.tunables.user_gutter_width)
+            .collect::<String>();
+        let user_gutter_empty_span: Span<'a> = Span::raw(user_gutter_empty);
```

Comment:
You can use the helper function from `util` here:

```suggestion
        let user_gutter_empty_span = space_span(self.settings.tunables.user_gutter_width, Style::default());
```